File: TDDMicroExercises.ProposedSolution/CSharp/UnicodeFileToHtmlTextConverter.SomeDependencies/aTextConverterClient2.cs
namespace TDDMicroExercises.OneSolution.UnicodeFileToHtmlTextConverter.SomeDependencies
{
    public class aTextConverterClient2
    {
		// A class with the only goal of simulating a dependency on UnicodeFileToHtmTextConverter
		// that has impact on the refactoring.

		UnicodeFileToHtmTextConverter.UnicodeFileToHtmTextConverter _textConverter;

        public aTextConverterClient2()
        {
            _textConverter = new TDDMicroExercises.OneSolution.UnicodeFileToHtmTextConverter.UnicodeFileToHtmTextConverter("anotherFilename.txt");

            var html = _textConverter.ConvertToHtml();
        }

    }
}

File: TDDMicroExercises/Java/TextConverter/src/main/java/tddmicroexercises/textconvertor/somedependencies/aTextConverterClient2.java
package tddmicroexercises.textconvertor.somedependencies;

import tddmicroexercises.textconvertor.UnicodeFileToHtmlTextConverter;

import java.io.IOException;

public class aTextConverterClient2
{
    // A class with the only goal of simulating a dependency on UnicodeFileToHtmTextConverter
    // that has impact on the refactoring.


    UnicodeFileToHtmlTextConverter _textConverter;

    public aTextConverterClient2() throws IOException
    {
        _textConverter = new UnicodeFileToHtmlTextConverter("anotherFilename.txt");

        String html = _textConverter.convertToHtml();
    }

}

